backend/src/main/scala/cromwell/backend/BackendJobExecutorActor.scala

```diff
@@ -0,0 +1,42 @@
+package cromwell.backend
+
+import akka.actor.{Actor, ActorLogging}
+import akka.event.LoggingReceive
+import cromwell.backend.BackendJobExecutorActor.{CleanUp, Execute, Prepare, Stop}
+
+object BackendJobExecutorActor {
+
+  // Commands
+  sealed trait BackendJobExecutorActorMessage
+  case object Prepare extends BackendJobExecutorActorMessage
+  case object Execute extends BackendJobExecutorActorMessage
+  case object Stop extends BackendJobExecutorActorMessage
+  case object CleanUp extends BackendJobExecutorActorMessage
+
+  //TODO: identify other generic events within job execution functionality.
+  // Events
+  sealed trait BackendJobExecutorActorEvent
+  case object PrepareCompleted extends BackendJobExecutorActorEvent
+  case class PrepareFailed(throwable: Throwable) extends BackendJobExecutorActorEvent
+  case object ExecutionStarted extends BackendJobExecutorActorEvent
+  case class ExecutionCompleted(result: String) extends BackendJobExecutorActorEvent
```

Comment:
I just defined that because I could not get to this point yet. I need to take a look to what we have right now and propose a new structure if needed.